roles/servicetelemetry/tasks/pre.yml

```diff
@@ -33,8 +33,11 @@
 
 - name: "Set supporting container image paths"
   set_fact:
-    prometheus_webhook_snmp_container_image_path: "{{ lookup('env', 'RELATED_IMAGE_PROMETHEUS_WEBHOOK_SNMP_IMAGE') | default('quay.io/infrawatch/prometheus-webhook-snmp:latest', true) }}" # noqa 204
+    alertmanager_image_path: "{{ lookup('env', 'RELATED_IMAGE_ALERTMANAGER_IMAGE') | default('quay.io/prometheus/alertmanager:latest', true) }}" # noqa 204
+    grafana_image_path: "{{ lookup('env', 'RELATED_IMAGE_GRAFANA_IMAGE') | default({{ servicetelemetry_defaults.graphing.grafana.base_image }}, true) }}" # noqa 204
```

Comment:
This doesn't appear to get used. I think a change is required here: https://github.com/infrawatch/service-telemetry-operator/blob/dd6e1b39b2759d8e276a6b7e55b95fb680bcf868/roles/servicetelemetry/templates/manifest_grafana.j2#L10

Reply:
Yea a bit more adjusting here required, but if we go with this approach, I'll make sure everything is functioning as intended.